src/constraints.jl

```diff
@@ -727,3 +727,313 @@ function list_of_constraint_types(model::Model)
     return Tuple{DataType, DataType}[(jump_function_type(model, f), s)
                                      for (f,s) in list]
 end
+
+
+"""
+    perturbation_range_of_feasibility(constraint::ConstraintRef)
```

Comment:
Maybe call this `lp_rhs_perturbation_range`? "Range of feasibility" could also be interpreted as the slack in the constraint, i.e., how much you can perturb the constraint to invalidate the current solution.